cv/CV-guide.md

```diff
@@ -0,0 +1,171 @@
+# Examples of cv term use for QC
+
+## Using existing term definitions to write a 
+### single value metric:
+If you would want to report the number of MS1 spectra for a particular run, you can use the cv Term
+```
+[Term]
+id: QC:4000111
+name: Number of MS1 spectra
+def: "Number of MS1 spectra" [PSI:QC]
+comment: A lower number of MS1 spectra acquired during one sample run compared to similar runs can indicate mismatched instrument settings or issues with the instrumentation.
+is_a: QC:4000003 ! single value
+is_a: QC:4000010 ! ID free
+is_a: QC:4000001 ! QC metric
+has_units: STATO:0000047! count
+```
+to define your metric object which would look like 
+```
+{
+    "accession": "QC:4000111",
+    "name": "Number of MS1 spectra",
+    "value": 7130,
+}
+```
+where the value would then be implicitly interpreted as of type `count`.
+
+### n-tuple:
+If we choose to write a n-tuple type, for example:
+```
+[Term]
+id: QC:4000113
+name: MZ acquisition range
+def: "Upper and lower limit of m/z values at which spectra are recorded." [PSI:QC]
+comment: Acquisition levels can be used as a criterion to assess the comparability of instrument settings between runs. 
+is_a: QC:4000010 ! ID free
+is_a: QC:4000001 ! QC metric
+is_a: QC:4000004 ! n-tuple
+has_units MS:1000040 ! m/z
+```
+Then we can assign the tuples as an n-array and their units implicitly assumed to be of `m/z` unit like this:
+```
+{
+    "accession": "QC:4000113",
+    "name": "MZ acquisition range",
+    "value": [400,1200],
+}
+```
+
+
+### matrix:
+TBD
+
+### table:
+If we want to report multiple types of values for multiple observations, we will choose a table. A table will be defined by it's columns, which can be optional. Here is an example
+```
+[Term]
+id: QC:4000095
+name: Sampling rates
+definition: "The sampling rates of identified peptides and their respective frequency (accumulated by the sampling rate, i.e. how often peptides were sampled n times and how many peptides were resampled at that rate. Optionally, which peptides were sampled at that rate, space separated)" [PSI:QC]
```

Comment:
I dont understand this definition...
If its a `rate` it should be a rational number, but below, its simply counts.

And how can a peptide be sampled, e.g., 5 times, and that occurs 3 times? Is this metric accumulating data over many  runs?

Reply:
ok, now I get it.... sorry.
is this more clear: `In this table, each row contains three values: Sample rate 's', i.e. how often a peptide was sampled in a run; a count 'c', for how many peptides were sampled this often ('s' times); peptide sequences (a single string of exactly 'c' peptides). The sampling rate must be unique across all rows.`??
This is mainly about the CV term, not the PR per se though...